nasa/nos3
# [nasa/nos3#426] Resolve SBN Segfault when client not available

It was reported that SBN would segfault and then crash all of FSW should it not be able to create the connection to a client. The SBN code was updated to default to localhost should hostname resolution fail. 

How to test:
* In `./scripts/fsw/fsw_cfs_launch.sh` comment out OnAIR
* `make clean && make`
* `make launch`
* Confirm OnAIR not running (see simulator tabs) and that FSW does not crash

The following merge requests must be closes and submodules updated here prior to closure:
* https://github.com/nasa-itc/SBN/pull/1

Closes #426
